### PR TITLE
Translate Term classes

### PR DESCRIPTION
**Pull request by GasimGasimzada**
In this PR, I have replaced all the existing translations of classes with new term: sinif.

@Maharramoff Please check if there are any issues in translations, specifically the following:

* content/tutorial/tutorial.md:465
* content/tutorial/tutorial.md:574
* content/docs/state-and-lifecycle.md:79
* content/docs/state-and-lifecycle.md:106
* content/docs/state-and-lifecycle.md:307
* content/docs/reference-react-component.md:32
* content/docs/reference-react-component.md:34
* content/docs/reference-react-component.md:588
* content/docs/reference-react-component.md:620
* content/docs/optimizing-performance.md:224
* content/docs/optimizing-performance.md:294
* content/docs/optimizing-performance.md:332
* content/docs/react-without-jsx.md:43
* content/docs/handling-events.md:61
* content/docs/reference-dom-elements.md:31
* content/docs/react-without-es6.md:7:
* content/docs/react-without-es6.md:103:
* content/docs/components-and-props.md:35
* content/docs/hooks-state.md:103
* content/docs/hooks-state.md:133
* content/docs/hooks-faq.md:229
* content/docs/hooks-faq.md:261
* content/docs/hooks-faq.md:290
* content/docs/hooks-faq.md:716
* content/docs/hooks-faq.md:750
* content/docs/typechecking-with-proptypes.md:180
* content/docs/hooks-overview.md:63
* content/docs/hooks-effect.md:151
* content/docs/hooks-effect.md:343
* content/docs/refs-and-the-dom.md:65
* content/docs/reference-react.md:102
* content/docs/composition-vs-inheritance.md:172
* content/warnings/refs-must-have-owner.md:34
* content/warnings/legacy-factories.md:47

These files refer to translations of words such as "sinfi", "sinfə" etc.

You can reproduce this search by using the following command:

```bash
grep -Rn 'sinf' content/
```

There is one false positive but just ignore it :)

### CODE REVIEW

File: content/docs/reference-react-component.md [hunk 1]
```diff
@@ -585,7 +585,7 @@ Adi halda, siz `render()`-də `forceUpdate()`-i heç bir halda işlətməməli v
 
 ### `defaultProps` {#defaultprops}
 
-`defaultProps` klasa ilkin propların verilməsi üçün komponent klasının parametri kimi təyin edilir. Bu `undefined` proplar üçün işlənilir. `null` proplarda işləmir. Məsələn:
+`defaultProps` parametri sinifə ilkin propların verilməsi üçün komponent sinfinin parametri kimi təyin edilir. Bu `undefined` proplar üçün işlənilir. `null` proplarda işləmir. Məsələn:
```
Comment (Maharramoff) [inline content/docs/reference-react-component.md]: @GasimGasimzada  sinfə

File: content/docs/refs-and-the-dom.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Applikasiyada "işləri həll etmək" üçün ref-lərdən istifadə etmək ist
 
 ### Ref-lərin Yaradılması {#creating-refs}
 
-Ref-lər `React.createRef()` funksiyası ilə yaranaraq `ref` atributu ilə React elementlərinə qoşulur. Adətən, ref-lərə komponentin hər yerindən istinad edilə bilməsi üçün ref-lər komponent yaradılan zaman komponent klasının instansiya parametrinə təyin edilir.
+Ref-lər `React.createRef()` funksiyası ilə yaranaraq `ref` atributu ilə React elementlərinə qoşulur. Adətən, ref-lərə komponentin hər yerindən istinad edilə bilməsi üçün ref-lər komponent yaradılan zaman komponent sinifinin instansiya parametrinə təyin edilir.
```
Comment (Maharramoff) [inline content/docs/refs-and-the-dom.md]: sinfinin

File: content/docs/refs-and-the-dom.md [hunk 3]
```diff
@@ -161,9 +161,9 @@ class Parent extends React.Component {
 }
 ```
 
-Bu komponentə ref lazım olduqda komponenti klasa çevirin (state və ya lifecycle metodları lazım olduğu kimi).
+Bu komponentə ref lazım olduqda komponenti sinifə çevirin (state və ya lifecycle metodları lazım olduğu kimi).
```
Comment (Maharramoff) [inline content/docs/refs-and-the-dom.md]: sinfə

File: content/docs/state-and-lifecycle.md [hunk 4]
```diff
@@ -72,13 +72,13 @@ Bunu yerinə yetirmək üçün `Clock` komponentinə "state" əlavə etmək laz
...
+## Funksiyanın sinifə çevrilməsi {#converting-a-function-to-a-class}
 
-`Clock` kimi funksional komponenti klas komponentinə  5 addımda çevirmək olar:
+`Clock` kimi funksional komponenti sinif komponentinə  5 addımda çevirmək olar:
 
-1. İlkin komponentlə adı eyni olan, `React.Component` klasını genişləndirən [ES6 klası](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes) yaradaq.
+1. İlkin komponentlə adı eyni olan, `React.Component` sinfini genişləndirən [ES6 sinfi](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes) yaradaq.
 
-2. Bu klasa `render()` adlı boş metod əlavə edək.
+2. Bu sinifə `render()` adlı boş metod əlavə edək.
```
Comment (Maharramoff) [inline content/docs/state-and-lifecycle.md]: sinfə

File: content/docs/state-and-lifecycle.md [hunk 5]
```diff
@@ -245,7 +245,7 @@ Bu metodlara "lifecycle metodları" deyilir.
 
 Diqqət yetirsək, taymerin ID-sini `this` də saxladığımızı görərsiniz (`this.timerID`).
 
-`this.props` React tərəfindən quraşdırılır, `this.state`-in də xüsusi anlamı var. Bu ikisindən savayı klasa hər hansı məlumat saxlamaq üçün məlumat axımında iştirak etməyən başqa verilənlər əlavə etmək olar (taymerin id-si kimi).
+`this.props` React tərəfindən quraşdırılır, `this.state`-in də xüsusi anlamı var. Bu ikisindən savayı sinifə hər hansı məlumat saxlamaq üçün məlumat axımında iştirak etməyən başqa verilənlər əlavə etmək olar (taymerin id-si kimi).
```
Comment (Maharramoff) [inline content/docs/state-and-lifecycle.md]: sinfə

**Comment by GasimGasimzada** [comment]
Thank you! The issues have been resolved. I am merging this one and closing #163.

**Pull request merged**